pages/1.11/installing/development/index.md

```diff
@@ -1,13 +1,12 @@
 ---
 layout: layout.pug
-navigationTitle:  Local
-title: Install DC/OS Locally
-menuWeight: 2
+navigationTitle: Development
+title: Development
+menuWeight: 5
 excerpt: Automating local installation of DC/OS for development and testing
-enterprise: true
 ---
 
-Two tools automate the local installation of DC/OS for development and testing:
+The following tools automate the local installation of DC/OS for development and testing:
 
 - [DC/OS Vagrant](https://github.com/dcos/dcos-vagrant/) - DC/OS on virtual nodes using Vagrant and VirtualBox
 - [DC/OS Docker](https://github.com/dcos/dcos-docker/) - DC/OS on containerized nodes using Docker
```

Comment:
DC/OS Docker is a deprecated project. DC/OS E2E CLI gives you much much better user experience.

https://dcos-e2e.readthedocs.io/en/latest/cli.html
https://github.com/dcos/dcos-e2e

Reply:
@bernadinm ??